dependencies.gradle

```diff
@@ -4,8 +4,8 @@
  */
 ext.versions = [
     minSdk: 23,
-    targetSdk: 28,
-    compileSdk: 28,
+    targetSdk: 29,
+    compileSdk: 29,
```

Comment:
This breaks having the store in external storage

Reply:
requires `android:requestLegacyExternalStorage=”true”`

Reply:
I see, that's why we haven't switched yet. Is there already a plan for what to do about this?

Reply:
I have a SAFUtils in v2 which we can use to implement SAF in v1.
`android:requestLegacyExternalStorage=”true”` should work in the meantime.

Reply:
Great!